py/server/tests/test_udf_args.py

```diff
@@ -405,9 +405,9 @@ def f(x: {p_type}) -> bool:  # note typing
                 t = empty_table(1).update(["X = i", f"Y = f(({p_type})X)"])
                 self.assertEqual(1, t.to_string(cols="Y").count("true"))
 
-
-        np_int_types = {"np.int8", "np.int16", "np.int32", "np.int64"}
-        for p_type in np_int_types:
+    def test_np_typehints(self):
+        widening_np_int_types = {"np.int32", "np.int64"}
```

Comment:
I see tons of numpy testing changes, but I don't see testing changes for widening with plain python types.  Make sure that those types are well covered in tests since we are now pushing users to them.